utils/simulation/cavity_service.py

```diff
@@ -134,6 +134,9 @@ class CavityPVGroup(PVGroup):
     step_temp: PvpropertyFloat = pvproperty(
         value=35.0, name="STEPTEMP", dtype=ChannelType.FLOAT
     )
+    tone_counts: PvpropertyFloat = pvproperty(
+        value=35.0, name="DAC_AMPLITUDE", dtype=ChannelType.FLOAT
```

Comment:
This needs to be updated to be the real PVs too... I think we can just have duplicates for each cavity so we don't have to think too hard. Like the Rack A cavities can have both RFS1A and RFS2A